shortcuts-disco-site/README.md

```diff
@@ -5,6 +5,12 @@
 npm run dev
 ```
 
+If you're having error `next: Comamnd not found`:
+
+```bash
+npm install -g next
```

Comment:
Are you sure that it needs to be global?

Based on the official docs you don't need it: https://nextjs.org/docs/getting-started/installation#manual-installation

Reply:
Yes `-g` is mistake, sorry.